packages/fether-react/src/Health/Health.js

```diff
@@ -54,29 +51,28 @@ class Health extends Component {
 
     const chainNameAppend = isLoading(chainName) ? '' : ` (${chainName})`;
 
-    switch (status) {
-      case STATUS.NO_NODE_CONNECTED_AND_NO_INTERNET:
-        return 'No internet. No nodes connected';
-      case STATUS.DOWNLOADING:
-        return `Downloading Parity Ethereum... (${payload.percentage}%)`;
-      case STATUS.LAUNCHING:
-        return 'Launching the node...';
-      case STATUS.NODE_CONNECTED_AND_NO_INTERNET:
-        return 'No internet. Connected to node.';
-      case STATUS.NO_CLOCK_SYNC:
-        return 'Clock of host not in sync';
-      case STATUS.NO_PEERS:
-        return 'No peer node connections';
-      case STATUS.SYNCING:
-        return `Syncing...${
-          payload && payload.percentage && payload.percentage.gt(0)
-            ? ` (${payload.percentage.toFixed(0)}%)`
-            : ''
-        }${chainNameAppend}`;
-      case STATUS.GOOD:
-        return `Synced${chainNameAppend}`;
-      default:
-        return JSON.stringify(payload); // Just in case payload is an object
+    if (!status.nodeConnected && !status.internet) {
+      return 'No internet. No nodes connected';
+    } else if (status.downloading) {
```

Comment:
There needs to be an additional case when we are connected to the internet, but not to our local node: "Not connected to light node" or the current "Connecting to node..."

Reply:
I've pushed a commit making it "Connecting to node..."